test/error-message-lister.js

```diff
@@ -0,0 +1,242 @@
+        relatedSourceCode: 'code',
+        action:            'testAction'
+    },
+    {
+        code:              TYPE.invisibleActionElement,
+        stepName:          'Step',
+        relatedSourceCode: 'code',
+        action:            'test-action',
+        element:           'element'
+    },
+    {
+        code:              TYPE.incorrectDraggingSecondArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code',
+        action:            'action'
+    },
+    {
+        code:              TYPE.incorrectPressActionArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.emptyTypeActionArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:     TYPE.unexpectedDialog,
+        stepName: 'Step',
+        dialog:   'test-dialog',
+        message:  'message'
+    },
+    {
+        code:     TYPE.expectedDialogDoesntAppear,
+        stepName: 'Step',
+        dialog:   'test-dialog'
+    },
+    {
+        code:              TYPE.incorrectSelectActionArguments,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.incorrectWaitActionMillisecondsArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.incorrectWaitForActionEventArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.incorrectWaitForActionTimeoutArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.waitForActionTimeoutExceeded,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.emptyIFrameArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.iframeArgumentIsNotIFrame,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.multipleIFrameArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.incorrectIFrameArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    },
+    {
+        code:              TYPE.uploadCanNotFindFileToUpload,
+        stepName:          'Step',
+        relatedSourceCode: 'code',
+        filePaths:         ['path1', 'path2']
+    },
+    {
+        code:              TYPE.uploadElementIsNotFileInput,
+        stepName:          'Step',
+        relatedSourceCode: 'code',
+        filePath:          'path'
+    },
+    {
+        code:              TYPE.uploadInvalidFilePathArgument,
+        stepName:          'Step',
+        relatedSourceCode: 'code'
+    }
+];
+
+var testMock = {
+    name:    'fixtureTest',
```

Comment:
It would be nice to spread errors between different tests in different fixtures and to have at least a couple of successful tests to cover more possible layouts.